jira/client.py

```diff
@@ -196,7 +197,7 @@ class JIRA(object):
     AGILE_BASE_URL = GreenHopperResource.AGILE_BASE_URL
 
     def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=None, kerberos=False,
-                 validate=False, get_server_info=True, async=False, logging=True, max_retries=3, proxies=None):
+                 validate=False, get_server_info=True, async=False, max_retries=3, proxies=None):
         """Construct a JIRA client instance.
```

Comment:
-1 on introducing `**kwargs` for the reasons you mentioned above.
What is your goal?
Take `logging` without breaking existing calls, but ignore it?
Take `logging` and keep the current behaviour? (controls one log line in `create_project()`)
Take `logging` and change it to control all logging?

I don't understand the point of having the argument. If we're using the python `logging` framework correctly then calling code can customising what logging they do and don't get using features of the `logging` library...